frontend/src/metabase/actions/components/ActionViz/ActionParameterMapping.tsx

```diff
@@ -61,7 +61,7 @@ export const ActionParameterMappingForm = ({
   }, [action]);
 
   const handleParameterChange = useCallback(
-    (dashboardParameterId, target) => {
+    (dashboardParameterId: any, target: ParameterTarget) => {
```

Comment:
The component that we pass `handleParameterChange` to calls for the first value to be `string | null`, and it explicitly passes null back under a couple of situations (ask the user or default value). However, `setParameterMapping()` does not expect null values, and updating that requires updating a number of functions that it passes values to. I'm not familiar enough with this area of the codebase to be confident in these changes, so I'm setting this to `any` for the time being